### PR TITLE
Prepare 0.3.3 release

### PR DESCRIPTION
**Pull request by christofdamian**
Add support for SoundCloud embedding

### CODE REVIEW

**Comment by ifosch** [comment]
+1

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,4 +1,6 @@
 # Changelog
+## 0.3.3 2018/4/12
```
Comment (nicolas-fricke) [inline CHANGELOG.md]: Minor thing, but to keep it aligned with the other entries a `-` between the version and the date would be nice

```md
## 0.3.3 - 2018/04/12
```

**Comment by bbrassart** [comment]
+1

**Review by nicolas-fricke — APPROVED** [review]
Thanks for the change

File: CHANGELOG.md (hunk 1 above)
Comment (christofdamian) [reply CHANGELOG.md]: done

**Pull request merged**